docs/protocol/specifications/vm1.md

```diff
@@ -75,8 +75,8 @@ Execution of any of the following instructions results in immediate failure:
 * [AMOUNT](#amount)
 * [MINTIME](#mintime)
 * [MAXTIME](#maxtime)
-* [TXREFDATAHASH](#txrefdatahash)
-* [REFDATAHASH](#refdatahash)
+* [TXDATAHASH](#txdatahash)
```

Comment:
I don't mean make it a varstring; I mean keep it as a 32-byte string but call it something other than a hash. A `string32` for instance (which happens to be suitable for storing 256-bit hashes).